source/Calamari.Tests/Fixtures/Certificates/WindowsX509CertificateStoreFixture.cs

```diff
@@ -176,6 +176,24 @@ public void CanImportCertificateForUser()
 
             sampleCertificate.EnsureCertificateNotInStore(storeName, StoreLocation.CurrentUser);
         }
+        
+        [Test]
+        public void CanImportCertificateWithNoPrivateKeyForSpecificUser()
+        {
+            // This test cheats a little bit, using the current user 
+            var user = WindowsIdentity.GetCurrent().Name;
+            var storeName = "My";
+            var sampleCertificate = SampleCertificate.CertWithNoPrivateKey;
+
+            sampleCertificate.EnsureCertificateNotInStore(storeName, StoreLocation.CurrentUser);
+
+            WindowsX509CertificateStore.ImportCertificateToStore(Convert.FromBase64String(sampleCertificate.Base64Bytes()), sampleCertificate.Password,
+                user, storeName, sampleCertificate.HasPrivateKey);
+
+            sampleCertificate.AssertCertificateIsInStore(storeName, StoreLocation.CurrentUser);
+
+            sampleCertificate.EnsureCertificateNotInStore(storeName, StoreLocation.CurrentUser);
```

Comment:
How much do we care about wrapping this in a try/finally? This way we know the certificate was removed, or at least we tried, even if the test failed?